Infra-Design.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2021-05-12T22:30:07.588Z" agent="5.0 (Windows)" etag="ubDmKSnVXtSJsPqbGop4" version="14.6.12" type="device"><diagram id="HsWXBNt8ZrJyBVVjh0Ag" name="Page-1">7Vxbc5s4FP41fowHSYjLY31Jm5nutDPZzrb70sFGtmmw5QUSO/31K0DCIMmG4mtiJ+kUHSQhdL7znSMd2R3Un68/Rt5y9hf1SdiBhr/uoEEHQmBA2En/DP+VS7Dt5pJpFPhcthE8Br+JaMqlz4FP4krFhNIwCZZV4ZguFmScVGReFNFVtdqEhtWnLr0pUQSPYy9Upf8EfjLLpQ60N/JPJJjOxJOBxd9v7onK/E3imefTVUmEhh3UjyhN8qv5uk/CdPbEvOTt7rfcLQYWkUXSpMG/L+jL8Onx8dcYP3yLHP/zkP59x3t58cJn/sJ8sMmrmAH6nITBgvSLCTY6qDehi6RPQxpldRD7vU+f2ptGnh+Qzb0FXZC0ehCGuupxEtEnIlX2vXhGfP6gFxIlAVPHZ29Ewq80DpKALti9EU0SOi9V+BAG0/RGQpdM6vHSmI2FsL57s2QesjLgY+dAA1CU+fumj/TiZf6ik2CdjqO3pEHay/CFdRbzTpg2l2mD+XqaIr/rrWKzOw6DTBk9VTdiotloybok4rr6SOicJNErq8Lv2hw23HBMs2vjXLLa4NB2ctGsBEHb4ujnyJ8WXW/AwS44Pv4AK7AlVo6tT40qUDdiQyE/MVLBd48djKztWD2A7hCSlGeomjO6WNWdg4X04NpD9dojPqM+XqRRMqNTuvDC4UZannWy8D+k/JrqLqTjp1QUjrKyUGFm4F6UiHp8glnL+yAstFehAKYfbOKeZbI7TIULvwAO00b0+j0tdKEryj+yMhbFwbpcefBaLn0lUcBmMgVPJtyq5pg+R2NSz5nszaYkqbeXdFZ3gqaECmBwpEQk9JLgpeqEdJjg3X1NKaoEQFwFIBT9ii7yl+Styk5D6sis6yifBaWjDKLFO7ZHrfnGUdseZbAhygA4H8xMo2AxEd/ZLYGGTT1lnghnQAc0K0x9uR+8sMtperlarbqjwOuORqOJ90S6YzrvjiJRkT24VFfTXIiyJhWJ3FQD8cxhVsGsOEXZd84D388sICJx8NsbFSFOFtFkM4t7HTzQAnWnPcqusAjA+UMqMa7ORTLgIOxWVC4g3xaUogqdTGJyFJTg98pGtSxjXRLJuG05RgrLgNzRFpJhc++9lqpx69k6Ygwd3Xg3YMw7PCg07Sb81X+4cvoSBnwA+jJNiKr0ZV46fVn19HU5C7jQm498b8vyzTzq8s10t/BEef1m6dZvLtqu7r1Wb8579TyHi4ON87koBCTAoLbLLeRKvg7KXR3IRykrRGTsHBlCzq76x3FqbkOnFj57iyv2aoIcrtOrCdu7ubW6XUlDMnn77G4NNMg/VM2ix8bcz/8xE4H9VNKFWCPUyWxVCNRq7D+ge4Is1MlsVQjUamlJjLoq1MlsrI5Ybg00rYHUOuOUvdI5KVBtZ2iUgToIItZRbi2LNPBQwT0wcB/YunBhkv2cwBj/OOmjTfAwys48+cM4HU/K4PlVtRZ5ig9jsRjDKhkXAUtdDsg5lsHCNx6IknWQ5FkEzEs/Snc2CYS08FoqHC590DigRWcMaKUV0J3ldqFR+pGyVI13YKAc3zIXAI8T4FrSg+zdAS50jP3qAwccPyIGui1I7T71lcfEBVEdICgGUIISvPiYuEG29xZX3eKqK42rkCVxd3FI62xx1Zvam438WAUjHNqo5xx1BWtjWW+GqeoNqnpzdpDzfnrT5V3eUjzcPo4VPqYcyM7xt88PX9B3r/e7N/71E36agfXdGeNYCyhBWsudWbu2p2OfULjeFIDYJbpopJmmhA8TtESaISPNdeVTgcfGmm7j/eaNZG8EzK60MjAtTRxxSn8k4phrZAmjIUu452MJJevW9iiLci6z4VGWQzFEYS3XhzOrIcycM8LMsaRtNgQt2Yc0hpq0F1g6j3V4sGlnsu2HVM7ijeL/NN5o4H6wTVDnjTR7VQpAd3gjWGUErHFFR8vuafUG6/X2BjhCMuBa0tgF4fpNf/d0JCFtgqC2R7fNuo6OTA8N9jpv9CCffb1jbCDI4HwM8T4+3nEQhmi6m4LPRhB3bfdOZIJQOjoyQTQ4tn85BLHfAaG9OMKS9YQMdG6GaLAvfi0MgZvGENbJKMIC8kLjDtgt970sW/ZQprJmOTJRNNhivWVNb1nTt5s13c89GLKxA2EgJ0ib7trKuxnszWBvBqt+jsmu5qSQpQnnTmmuTY7qX0s05zRc750whanwO4LK95E0juZMubNiQ/JUu8bvI0VxELC5Fwc2rKQoIG6dorCgEpnIGdbWYGPFzXd05dU3X3WGhv8D</diagram></mxfile>
```

Comment:
Where to test this file?

Reply:
Save this file on computer.
On https://app.diagrams.net/  , using File -> Import From -> Device
Select this file and the diagram will be showed